src/constants/update-details-journeys.ts

```diff
@@ -7,14 +7,17 @@ const SUPPORTED_COMBO_EVENTS = Object.freeze({
   UPDATE_CANCEL: "cancel",
 });
 
-const UNSUPPORTED_COMBO_EVENTS = Object.freeze({
+export const UNSUPPORTED_COMBO_EVENTS = Object.freeze({
   address: "address",
   givenNames: "given",
   familyName: "family",
   dateOfBirth: "dob",
-});
+}) satisfies Record<UpdateDetailsOptions, string>;
```

Comment:
What's this for?

Reply:
ah good spot, removing this - I think this was from trying to more strictly type `details` on line 248 in the ipv middleware but opted for just casting it as `UpdateDetailsOptions` instead since, at this point we can be more confident it's of type `UpdateDetailsOptions` given we're handling the "cancel" option already above it